libpoolprotocols/getwork/EthGetworkClient.cpp

```diff
@@ -77,7 +77,9 @@ void EthGetworkClient::disconnect()
 {
     // Release session
     m_connected.store(false, memory_order_relaxed);
-    m_conn->addDuration(m_session->duration());
+    if (m_session) {
```

Comment:
Preferably use clang-format to format your changes. Description here: https://github.com/ethereum/aleth/blob/master/CONTRIBUTING.md#aleth-contributing-and-review-guidelines.

Reply:
fixed in https://github.com/ethereum-mining/ethminer/pull/1824/commits/eb95ad3ff0c00561f54b881424141c031a677da2